packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row_variants.story.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <dt-stack gap="600">
+    <div>
+      <h3 class="d-mb8">
+        Ai Contact Centers leftbar
+      </h3>
+      <dt-recipe-contact-centers-row
+        description="Ai Contact Centers"
+      >
+        <template #right>
+          <dt-button
+            importance="clear"
+            :class="[
+              'leftbar-call-centers-duty-status__button',
+              'd-bar-pill',
+              'd-py4',
+              'd-fc-success',
+            ]"
+            class="d-to-ellipsis"
+            size="sm"
+            data-qa="leftbar-call-centers-duty-value"
+            @click.stop
+          >
```

Comment:
The button color on hover is not right, add `kind="muted"` to fix

![Screenshot 2024-05-01 at 12 43 01 p m](https://github.com/dialpad/dialtone/assets/87546543/0acb55c4-46ec-4b56-8d4f-aea2434eca03)

```suggestion
          <dt-button
            importance="clear"
            kind="muted"
            :class="[
              'leftbar-call-centers-duty-status__button',
              'd-bar-pill',
              'd-py4',
              'd-fc-success',
              'd-to-ellipsis',
            ]"
            size="sm"
            data-qa="leftbar-call-centers-duty-value"
            @click.stop
          >
```

Do the same with all the similar examples here.